### PR TITLE
E2E fixes after countryArea changes

### PR DESCRIPTION
**Pull request by szczecha**
I want to merge this change because it fixes the tests where we add or update addresses
Fixes: https://github.com/saleor/saleor-dashboard/issues/2904

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://automation-dashboard.staging.saleor.cloud/graphql/
MARKETPLACE_URL=https://apps.saleor.io
SALEOR_APPS_ENDPOINT=https://apps.saleor.io/api/saleor-apps

### Do you want to run more stable tests?
To run all tests, just select the stable checkbox. To speed up tests, increase the number of containers. Tests will be re-run only when the "run e2e" label is added.

1. [x] stable
2. [ ] giftCard
3. [ ] category
4. [ ] collection
5. [ ] attribute
6. [ ] productType
7. [ ] shipping
8. [ ] customer
9. [ ] permissions
10. [ ] menuNavigation
11. [ ] pages
12. [ ] sales
13. [ ] vouchers
14. [ ] homePage
15. [ ] login
16. [ ] orders
17. [ ] products
18. [ ] app
19. [ ] plugins
20. [ ] translations
21. [ ] navigation
22. [ ] variants
23. [ ] payments

CONTAINERS=4

### CODE REVIEW

**Review by krzysztofzuraw — APPROVED** [review]

**Review by andrzejewsky — APPROVED** [review]

File: cypress/fixtures/addresses.json [hunk 1]
```diff
@@ -37,6 +38,7 @@
     "postalCode": "37745",
     "country": "US",
     "countryArea": "TN",
+    "countryAreaFullName": "Tennessee",
```
Comment (karola312) [inline cypress/fixtures/addresses.json:41]: Why adding new filed "countryAreaFullName", not just change "countryArea" value from "TN", to "Tennessee"? In these way, we don't have to change anything in fillUpBasicAddress and in expectCorrectBasicAddress commands

Comment (szczecha) [reply cypress/fixtures/addresses.json:41]: Right now country area is a select type so I need to make changes in fillUpBasicAddress and use fillAutocompleteSelect there.

I was thinking about only changing the country area from code like "TN" to full name "Tennessee" but when I ran tests I noticed that it fails on the assertion for the country area - the backend returns the country area as code (This is not a bug, it's just data normalization on BE)
image

Comment (karola312) [reply cypress/fixtures/addresses.json:41]: In customer it work in other way. The backend returns the country area as full name. So tests for adding address in customer are not working after this changes. I think API should return the same value type in all places

Comment (karola312) [reply cypress/fixtures/addresses.json:41]: In customer it work in other way. The backend returns the country area as full name. So tests for adding address in customer are not working after this changes. I think API should return the same value type in all places

Comment (szczecha) [reply cypress/fixtures/addresses.json:41]: You're right, I asked the backend team yesterday about this case and they confirmed that they should return the code here as in other mutations with the address: https://github.com/saleor/saleor/issues/11572

**Pull request merged**